src/components/App/index.tsx

```diff
@@ -2,22 +2,33 @@ import { Typography } from '@ht6/react-ui';
 import '@ht6/react-ui/dist/styles/index.css';
 import cx from 'classnames';
 import { ToastBar, Toaster } from 'react-hot-toast';
-import { Route, Routes } from 'react-router-dom';
+import { Outlet, Route, Routes } from 'react-router-dom';
 
 import Application from '../../pages/Application';
 import Callback from '../../pages/Callback';
 import Dashboard from '../../pages/Dashboard';
+import Notion from '../../pages/Notion';
 import Layout from '../Layout';
 
 import styles from './App.module.scss';
 
 function App() {
   return (
-    <Layout>
+    <>
       <Routes>
-        <Route path='/' element={<Application />} />
-        <Route path='home' element={<Dashboard />} />
-        <Route path='callback' element={<Callback />} />
+        <Route path='notion' element={<Notion />} />
```

Comment:
Is there supposed to be two notion paths here?

Reply:
yes, the notion path is for the renderer